xbmc/platform/darwin/osx/MediaKeys.mm

```diff
@@ -16,25 +16,33 @@
 #include "utils/log.h"
 
 #import <AppKit/AppKit.h>
+#import <Foundation/Foundation.h>
```

Comment:
Not sure what this is for. GCD facilities are in dispatch.h iirc

Reply:
This was because of dispatch_sync (GCD) but I'll try to restrict the include